components/token-card/index.tsx

```diff
@@ -1,19 +1,34 @@
 import Link from "next/link";
+import moment from "moment";
 
 import React from "react";
 import { SmallBaseIcon } from "@/public/icons";
 import LBClickAnimation from "../click-animation";
-import useTimeAgo from "@/hooks/useTimeAgo";
 import { ILBTokenCard } from "./types";
 
+const timeAgo = (date: string) => {
+  const now = moment();
+  const createdMoment = moment(date);
+  const daysThreshold = 7;
+
+  const diffInDays = now.diff(createdMoment, "days");
+
+  if (diffInDays >= daysThreshold) {
+    return createdMoment.format("DD/MM/YY");
+  } else {
+    return createdMoment.fromNow();
+  }
+};
```

Comment:
change this to

```js
if (diffInDays >= daysThreshold) {
    return createdMoment.format("DD/MM/YY");
  } 

   return createdMoment.fromNow();
  

```


no need to use if else if we're using the return statement

Reply:
Done